src/tools/quotes/dump-to-db.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable no-console */
+/* eslint-disable import/no-extraneous-dependencies */
+import csv from 'csv';
+import fs from 'fs';
+import path from 'path';
+import mongodb from 'mongodb';
+import { Quote } from './IQuote';
+
+const CONNECTION_STRING = 'mongodb://localhost:27017';
```

Comment:
Can you pull this from the .env? You can check `src/main.ts` for how to get dotenv to pull in the values. We already have MONGODB_URI as an env variable.

Reply:
I'll try, I tried using the services before but there were some build errors. I'll try accessing it directly or using `dotenv` instead maybe

Reply:
Yeah don't need to put the service in, can just use dotenv to load the values and keep the mongo connection as you have.